include/swift/AST/SILOptions.h

```diff
@@ -162,6 +162,10 @@ class SILOptions {
   /// User code should never be compiled with this flag set.
   bool SILSerializeVTables = false;
 
+  /// If set, perform SIL serialization of a module after
+  /// high-level optimizations.
+  bool SILSerializeAfterHighLevelOptz = false;
```

Comment:
Nitpick: "Opts" or "Optzns" or "Optimizations". We don't use "Optz" anywhere else in the compiler except PassManager.cpp, and there only in one place.